sass/routes/_settings.scss

```diff
@@ -235,13 +235,26 @@
                     @include legacy-pie-clearfix;
                     @include alpha-property('background', $black, .05);
                     margin-bottom: 0;
-                    padding: 1em 0 1.3em;
+                    padding: 1em 1em 1.3em;
```

Comment:
Actually, this is probably the change that's causing the differences I showed above. Did you mean to change this padding?